### ISSUE TITLE
Problem with single dash

### ISSUE THREAD

**Issue by soelen** [acusticdemon/gulp-sass-import-json#7]
Greetings, I am defining keys for possible prefixes in my json file.
When I put a single - character as a value I am getting following issue: 

<img width="178" alt="grafik" src="https://user-images.githubusercontent.com/1114549/36542210-71825052-17e0-11e8-9bac-55c289db73c0.png">

<img width="802" alt="grafik" src="https://user-images.githubusercontent.com/1114549/36542218-7cfaa286-17e0-11e8-845a-d08d6f58c4ad.png">

But any other character seems to be working.

**Comment by acusticdemon** [comment]
Can you change - to '-'?

**Comment by soelen** [comment]
Well, that's interesting. Wrapping the dash with single quotes does work like a charm when I compile the sass code:

<img width="242" alt="grafik" src="https://user-images.githubusercontent.com/1114549/36660988-44304ada-1ada-11e8-815f-8467e6202269.png">

Unfortunatly the json file feeds data to template files as well:

<img width="418" alt="grafik" src="https://user-images.githubusercontent.com/1114549/36661037-6afa64d4-1ada-11e8-85c7-c39c11b8cd2b.png">

However I just realized I filled an issue on the wrong repository, this issue was ment for the sass-import-json team. Sorry for the trouble I have caused; I will close this issue immediately.

Thank you for your quick support and have a nice day!

Edit: https://github.com/Updater/node-sass-json-importer/issues/54
